### PR TITLE
[16.0][OU-ADD] l10n_es: Nothing to do

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT42251

@pedrobaeza please review!

### CODE REVIEW

File: docsource/modules150-160.rst [hunk 1]
```diff
@@ -300,7 +300,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | l10n_eg_edi_eta                                 |                      |No DB layout changes.                            |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| l10n_es                                         |                      |                                                 |
+| l10n_es                                         |                      |Nothing to do                                                 |
```
Comment (legalsylvain) [inline docsource/modules150-160.rst]:
```suggestion
| l10n_es                                         |                      |Nothing to do                                    |
```

Comment (stefan-tecnativa) [reply docsource/modules150-160.rst]: I didn't noticed it. Thanks!

**Review by legalsylvain — APPROVED** [review]

**Review by chienandalu — COMMENTED** [review]
Depends on

- [x] https://github.com/OCA/OpenUpgrade/pull/4070
- [x] https://github.com/OCA/OpenUpgrade/pull/3870
- [x] https://github.com/OCA/OpenUpgrade/pull/3871

**Comment by legalsylvain** [comment]
/ocabot migration l10n_es

<dependency></dependency>

**Comment by stefan-tecnativa** [comment]
@pedrobaeza All the modules it depended on have been merged. Is it possible to merge the PR?

**Review by chienandalu — APPROVED** [review]

File: docsource/modules150-160.rst [hunk 2]
```diff
@@ -300,7 +300,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | l10n_eg_edi_eta                                 |                      |No DB layout changes.                            |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| l10n_es                                         |                      |                                                 |
+| l10n_es                                         |                      |Nothing to do                                    |
```
Comment (pedrobaeza) [inline docsource/modules150-160.rst]: You have put it the wrong place:

```suggestion
| l10n_es                                         | Nothing to do        |                                               |
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
And rebase over latest changes.

**Comment by stefan-tecnativa** [comment]
@pedrobaeza thanks for the review and the comments!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**